pkg/cloudprovider/provider/anexia/resolve_config.go

```diff
@@ -126,6 +127,13 @@ func (p *provider) resolveNetworkConfig(log *zap.SugaredLogger, config anxtypes.
 	return &ret, nil
 }
 
+func (p *provider) resolveBandwidthLimitConfig(config anxtypes.RawConfig) (int, error) {
+	if config.BandwidthLimit != 0 && config.BandwidthLimit != 1000 && config.BandwidthLimit != 10000 && config.BandwidthLimit != 100 {
+		return 0, fmt.Errorf("Configuration for bandwidthLimit is not valid (0,100,1000,10000)")
+	}
+	return config.BandwidthLimit, nil
```

Comment:
Would love it if we could sort the bandwidth limits ascending. Suggestion (with a switch statement to make it easier to read):


```suggestion
    switch config.BandwidthLimit {
    case 0, 100, 1000, 10000:
    	return config.BandwidthLimit, nil
    default:
		return 0, fmt.Errorf("Configuration for bandwidthLimit is not valid (0,100,1000,10000)")
	}
```